beacon-chain/db/filesystem/pruner.go

```diff
@@ -99,10 +99,7 @@ func (p *blobPruner) warmCache() error {
 		}
 		p.Unlock()
 	}()
-	if err := p.prune(0); err != nil {
-		return err
-	}
-	return nil
+	return p.prune(0)
```

Comment:
@jsvisa I went into deepsource and overrode the complaint about this file, so that should no longer impact your other PR.

Reply:
I apologize for the late reply. I'm not quite understanding, are you saying that you have another branch also modifying this section of code, and the logic inside it happens to be needed here(after the success of `p.prune(0)`, there are more tasks to check)? If that's the case, there might be a conflict, is that correct?

Reply:
That's right, I'm working on a PR that will significantly change this file, making these changes redundant + conflicting, so it would be more convenient if we could just exclude this from the PR.

Reply:
thanks for the explain, reverted